.github/workflows/cicd.yaml

```diff
@@ -4,396 +4,83 @@ on:
   push:
     branches:
       - master
+      - develop
```

Comment:
Why this addition? To have the image:develop image "published"?

Reply:
that and upload security artifacts on all protected branches to ensure we can understand the state of security. I could however drop the integration tests bc those already ran on both branches and were a duplicate during release in any case